allRoutes/chRoutes.js

```diff
@@ -0,0 +1,111 @@
+const express = require('express');
+const knex = require('knex');
+const knexConfig = require('../knexfile');
+const db = knex(knexConfig.development);
+console.log(knexConfig);
+const router = express.Router();
+
+//++++++++++++++++++++++++++++++++++++++++++
+//   -   ALL ROUTES     -
+//++++++++++++++++++++++++++++++++++++++++++
+// - Get Endpoints. 
+//++++++++++++++++++++++++++++++++++++++++++++
+
+router.get('/', (req, res) => {
+    console.log('get all request working');
+    db('cohorts')
+        .then(allcohorts => {
+            if (allcohorts) {
+                res.status(200).json(allcohorts);
+            } else {
+                res.status(404).json({error: 'error', err});
+            }
+        })
+        .catch(err => {
+            res.status(500).json({error: 'error the cohorts info could not be retrieved', err})
+        });
+});
+
+//  /:id
+
+router.get('/:id', (req, res) => {
+    const { id } = req.params;
+    db('cohorts').where({ id }).first().then( thiscohort => {
+        if (thiscohort) {
+            res.status(200).json(thiscohort);
+        } else {
+            res.status(404).json({error: "error", err});
+        }
+    })
+    .catch(err => { res.status(500).json({error: 'error', err})});
+});
+
+
+//++++++++++++++++++++++++++++++++++++++++
+// - Post Endpoint.     -
+//++++++++++++++++++++++++++++++++++++++++
+
+router.post('/', (req, res) => {
+    console.log('post request working');
+    const content = req.body;
+    db('cohorts')
+        .insert(content)
+        .into('cohorts')
```

Comment:
This `into` isn't needed since you're already specifying the table by passing it into `db`.